vespa-exporter.py

```diff
@@ -180,7 +180,7 @@ def get_container_metrics(hostport):
             labelvalues = {}
             labelvalues['host'] = hostport
             if 'dimensions' in v:
-                for d in ['chain', 'handler', 'api', 'operation', 'status', 'serverName', 'serverPort']:
+                for d in ['chain', 'handler', 'api', 'operation', 'status', 'serverName', 'serverPort', 'httpMethod', 'gcName']:
```

Comment:
I meant, why don't we also add here for example: `documenttype`, `field`, `rankProfile` or `docidPartition`? (those are just some dimensions I stumbled upon)

Reply:
oh, I see. Good point. 

I cannot remember why I limited the dimensions to this short list. I guess I was just so overwhelmed by the amount of metrics. 

There is probably no reason why we cant just add all found dimensions.

Reply:
Ok, I did not find more than those four for now, can you add them in this MR? And others you might find too! :)